### PR TITLE
[WIP] Fix formatting and multimodule project structure

### PR DESCRIPTION
**Pull request by sshepel**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

## Quick checklist

The following checklist helps ensure your PR is complete:

- [ ] Eclipse Contributor Agreement signed, and signed commits - see [IP Requirements](https://deeplearning4j.org/eclipse-contributors) page for details
- [ ] Reviewed the [Contributing Guidelines](https://github.com/eclipse/deeplearning4j/blob/master/CONTRIBUTING.md) and followed the steps within.
- [ ] Created tests for any significant new code additions.
- [ ] Relevant tests for your changes are passing.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -280,8 +284,10 @@
         <reflections.version>0.9.10</reflections.version>
         <breeze.version>1.0</breeze.version>
 
-        <javacpp.parser.skip>false</javacpp.parser.skip>     <!-- To skip header file parsing phase: -Djavacpp.parser.skip=true  -->
-        <javacpp.compiler.skip>false</javacpp.compiler.skip> <!-- To skip native compilation phase: -Djavacpp.compiler.skip=true -->
+        <javacpp.parser.skip>false
+        </javacpp.parser.skip>     <!-- To skip header file parsing phase: -Djavacpp.parser.skip=true  -->
+        <javacpp.compiler.skip>false
+        </javacpp.compiler.skip> <!-- To skip native compilation phase: -Djavacpp.compiler.skip=true -->
```
Comment (treo) [inline pom.xml:290]: Is this actually equivalent to the variant without the new line?

**Review by treo — COMMENTED** [review]
Looks like a good clean up overall.

You have removed quite a lot of plugin configuration and consolidated it in the main pom.xml, does it still work as expected?
Esp. the git-commit-id-plugin looks like it may not work that way - but I'm only guessing here, and haven't actually verified my hunch. Did you verify that this still works?

File: pom.xml (hunk 1 above)
Comment (sshepel) [reply pom.xml:290]: Hm, it has been done by Idea formatter because of long line... but we can go with new line too...

**Comment by sshepel** [comment]
> Looks like a good clean up overall.
> 
> You have removed quite a lot of plugin configuration and consolidated it in the main pom.xml, does it still work as expected?
> Esp. the git-commit-id-plugin looks like it may not work that way - but I'm only guessing here, and haven't actually verified my hunch. Did you verify that this still works?

I have managed to compare builds (local, with changes in this branch and builds on CI) for several modules, so far looks ok.

**Comment by sshepel** [comment]
Will reopen this PR against Konduit repo....

**Pull request closed without merging**